.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: python
 python:
-- '2.7'
+  - '2.7'
```

Comment:
All tests will ran by tox with python version, specified in related section of tox.ini. 2.7 python executable only using to run tox.